app/models/wakes/resource.rb

```diff
@@ -28,6 +28,11 @@ def to_s
     EOS
   end
 
+  def recalculate_facebook_count!
```

Comment:
@benhutton Sounds good. I'll remove the bang here. I wanted to convey that this has a side effect, but then AR::save has a side effect but not a bang. 

Regarding where to put this method, I had originally put this in the service object but moved it to the AR model. I imagine there might be instances where we might want to call `Wakes::Resource#recaculate_facebook_count` outside of the service object. What do you think?